src/pages/taesooKim/Login/Login.js

```diff
@@ -1,11 +1,86 @@
-import React, { Component } from "react";
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+import "../../../styles/common.scss";
+
+class LoginTaeSoo extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+      disabled: true,
+    };
+  }
+
+  activeLogin = () => {
+    this.state.id.indexOf("@") !== -1 && this.state.pw.length >= 5
+      ? this.setState({ disabled: false })
+      : this.setState({ disabled: true });
+  };
+
+  handleLogin = () => {
+    fetch("http://10.58.7.180:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log(result));
+  };
+
+  handleInput = e => {
+    const { value, id } = e.target;
+    this.setState({
+      [id]: value,
+    });
+    this.activeLogin();
+  };
+
+  goToMain = () => {
+    if (!this.state.disabled) {
+      this.props.history.push("/main-soo");
+    }
+  };
 
-class LoginTaeSoo extends Component {
   render() {
-    return <div>
-      안녕하세요?
-    </div>;
+    const { id, pw, disabled } = this.state;
+    return (
+      <section className="login">
+        <h1 className="title">Westagram</h1>
+        <div className="login-container">
+          <div id="login-form">
+            <input
+              id="id"
+              className="username"
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+              onChange={this.handleInput}
+            />
+            <input
+              id="pw"
+              className="password"
+              type="password"
+              placeholder="비밀번호"
+              onChange={this.handleInput}
+            />
+            <button
+              className={`btn ${
+                id.indexOf("@") !== -1 && pw.length >= 5 ? "active" : ""
+              }`}
```

Comment:
```suggestion
              className={`btn ${ isActive ? "active" : "" }`}
```
로직만 읽어서는 한눈에 무슨 뜻인지 알아보기 힘드니 render 안쪽에 변수로 선언해두고 위와 같이 사용하면 어떨까요?

Reply:
네네 수정하였습니다🐸